vendor.conf

```diff
@@ -21,7 +21,7 @@ github.com/docker/go-metrics                        b619b3592b65de4f087d9f16863a
 
 # etcd/raft
 go.etcd.io/etcd                                     faa0e3c45c80cc5cbc453b38d7a1637c5ee97b05 git://github.com/errordeveloper/etcd.git # fork of release-3.4 without the custom load balancer code to work with newer versions of gRPC
-github.com/coreos/etcd                              faa0e3c45c80cc5cbc453b38d7a1637c5ee97b05 git://github.com/errordeveloper/etcd.git # alias for github.com/rexray/gocsi
```

Comment:
not sure if the "cannot build with this alias" was related to that, but GitHub has deprecated support for `git://` protocol (you can use `https://` though, that should still work)